evm/interchainMultisig.js

```diff
@@ -0,0 +1,563 @@
+'use strict';
+
+const { ethers } = require('hardhat');
+const {
+    getDefaultProvider,
+    utils: { arrayify, keccak256, formatEther, formatBytes32String, hashMessage, recoverAddress },
+    Contract,
+    BigNumber,
+} = ethers;
+const { sortBy } = require('lodash');
+const { Command, Option } = require('commander');
+const {
+    printInfo,
+    printWalletInfo,
+    isNumber,
+    isValidCalldata,
+    printWarn,
+    isNonEmptyStringArray,
+    isNumberArray,
+    isValidAddress,
+    mainProcessor,
+    isValidDecimal,
+    prompt,
+    isBytes32Array,
+    getGasOptions,
+    isAddressArray,
+    saveConfig,
+} = require('./utils');
+const { addBaseOptions } = require('./cli-utils');
+const IInterchainMultisig = require('@axelar-network/axelar-gmp-sdk-solidity/interfaces/IInterchainMultisig.json');
+const IGateway = require('@axelar-network/axelar-gmp-sdk-solidity/interfaces/IAxelarGateway.json');
+const IGovernance = require('@axelar-network/axelar-gmp-sdk-solidity/interfaces/IAxelarServiceGovernance.json');
+const IInterchainTokenService = require('@axelar-network/interchain-token-service/interfaces/IInterchainTokenService.json');
+const ITokenManager = require('@axelar-network/interchain-token-service/interfaces/ITokenManager.json');
+const IOperator = require('@axelar-network/interchain-token-service/interfaces/IOperator.json');
```

Comment:
You can use [getContractJSON](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/evm/utils.js#L975) function instead of hardcoding these paths, better in case folder structure of other repos changes in the future